scripts/create_pelorus_operator

```diff
@@ -115,15 +121,23 @@ pushd "${destination_dir}" || exit 2
     # Correct operator version, to be latest +1
     OPERATOR_VERSION=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/${OPERATOR_ORG_NAME}/${OPERATOR_PROJECT_NAME}/tag/" | jq .tags[].name | head -1 | sed -e 's|\"||g')
     echo "INFO: Current operator version: ${OPERATOR_VERSION}"
-    NEW_OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | awk -F. -v OFS=. '{$NF += 1 ; print}')
+    if [ ! -z ${file_version} ] && [ ${file_version} != ${OPERATOR_VERSION} ]; then
+        echo "ERROR: file version ${file_version} does not match released version ${OPERATOR_VERSION}."
+        exit 2
+    fi
+    if $no_update; then
```

Comment:
@mpryc added the logic to not update version, but there are still false positives